cunumeric/linalg/linalg.py

```diff
@@ -148,7 +148,8 @@ def matrix_power(a: ndarray, n: int) -> ndarray:
     # Use binary decomposition to reduce the number of matrix multiplications.
     # Here, we iterate over the bits of n, from LSB to MSB, raise `a` to
     # increasing powers of 2, and multiply into the result as needed.
-    z = result = None
+    z: Union[ndarray, None] = None
+    result: Union[ndarray, None] = None
```

Comment:
TIL: not possible to manually annotate any variable in an `a = b = c` situation.